server/bets.js

```diff
@@ -0,0 +1,27 @@
+const { getConnection } = require('./db');
+
+module.exports = {
+  getAllBets: function(cb) {
+    getConnection().query(`
+        SELECT *
+        FROM bets
+      `, (err = null, results) => {
+        return cb(err, JSON.parse(JSON.stringify(results)));
+      }
+    )
+  },
+
+  createBet: function(tournament_id, bettor_id, player_id, amount, cb) {
+    getConnection().query(`
```

Comment:
Update this to be like matches: https://github.com/nickweinberg/smash-dash/blob/master/server/matches.js#L4

Basically mysql/node is smart enough to take that object and convert it into an insertable row with magic